scripts/badger/claim_hh.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main(destination="0x042B32Ac6b453485e357938bdC38e0340d4b9276"): # trops
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+    destination = GreatApeSafe(destination)
+
+    rewards = voter.contract(r.hidden_hand.rewards_distributor)
+
+    hh_url = f"https://hhand.xyz/reward/1/{voter.address}"
+    response = requests.get(hh_url)
+    data = response.json()["data"]
+
+    aggregate = {"tokens": [], "amounts": []}
+    for item in data:
+        aggregate["tokens"].append(item["token"])
+        aggregate["amounts"].append(item["claimMetadata"]["amount"])
+
+    voter.take_snapshot(tokens=aggregate["tokens"])
+    destination.take_snapshot(tokens=aggregate["tokens"])
+
+    rewards.claim(
```

Comment:
i will say given that we have this [method](https://github.com/Badger-Finance/badger-multisig/blob/main/great_ape_safe/ape_api/badger.py#L154) in the badger class. will advise to leverage and allow passing an address for where bribes will be claimed

Reply:
good idea

Reply:
updated in [7d83c21](https://github.com/Badger-Finance/badger-multisig/pull/726/commits/7d83c21c22e4cc69971be521696cc371a355d3d1), allows address to be passed into `claim_bribes_hidden_hands` but claims from gravi aura strat by default as to not break other scripts